channeldb/mp_payment.go

```diff
@@ -177,6 +178,18 @@ func (m *MPPayment) InFlightHTLCs() []HTLCAttempt {
 	return inflights
 }
 
+// GetAttempt returns the specified htlc attempt on the payment.
+func (m *MPPayment) GetAttempt(id uint64) (*HTLCAttempt, error) {
+	for _, htlc := range m.HTLCs {
+		if htlc.AttemptID == id {
+			// nolint: scopelint
```

Comment:
nit: could also be written as
```
for i, htlc := range m.HTLCs {
    ...
    return &m.HTLCs[i], nil
}
```
to avoid `nolint`

Reply:
fixed